src/pages/projectBacklog/components/SprintIssue.jsx

```diff
@@ -204,17 +204,41 @@ export default function SprintIssue({
         <Typography variant="body2" sx={{ padding: "6px 12px", color: "#64748b" }}>
           Change assignee to:{" "}
         </Typography>
+        {/* <MenuItem value="" >
```

Comment:
remove this commented code

Reply:
format code before push.
https://code.visualstudio.com/docs/editor/codebasics#:~:text=VS%20Code%20has%20great%20support,)%20%2D%20Format%20the%20selected%20text.